test/nodejs/search.js

```diff
@@ -1240,7 +1240,8 @@ describe('Gix Position Tests', function () {
         "Kraft clinches ski jump title with final-event win",
         "Kraft wins final Planica event and ski-jumping World Cup",
         "Germany, Norway neck-and-neck after the first series in Planica",
-        "aa aa aa aa aa bb aa aa aa aa aa cc aa dd"
+        "aa aa aa aa aa bb aa aa aa aa aa cc aa dd",
+        "kk " + Array(1023).join("xx ") + "ll mm nn"
```

Comment:
do you mean instead of Array(1023).join("xx ") ? Using the current code makes it easier to understand the bottom tests.